test/lib/testUtils.ts

```diff
@@ -519,3 +520,31 @@ export async function waitForEvent<TEvent>(
         clearTimeout(timeoutId);
     });
 }
+
+export async function deleteAllAttributes(client: ConnectorClient, clientAddress: string): Promise<void> {
+    const attributesResponse = await client.attributes.getAttributes({});
+    expect(attributesResponse).toBeSuccessful(ValidationSchema.ConnectorAttributes);
+
+    for (const attribute of attributesResponse.result) {
+        let result: ConnectorHttpResponse<any> = ConnectorHttpResponse.error({
+            code: "",
+            message: "No delete method called",
+            docs: "",
+            time: "",
+            id: ""
+        });
+        if (!attribute.shareInfo) {
+            result = await client.attributes.deleteRepositoryAttribute(attribute.id);
+        }
+        if (attribute.shareInfo && attribute.content.owner === clientAddress) {
+            result = await client.attributes.deleteOwnSharedAttributeAndNotifyPeer(attribute.id);
+        }
+        if (attribute.shareInfo && attribute.content.owner !== clientAddress) {
+            result = await client.attributes.deletePeerSharedAttributeAndNotifyOwner(attribute.id);
+        }
+        if (attribute.shareInfo?.thirdPartyAddress) {
+            result = await client.attributes.deleteThirdPartyRelationshipAttributeAndNotifyPeer(attribute.id);
+        }
+        expect(result.error).toBeUndefined();
```

Comment:
1. early return or else ifs?
2. this is not how we work with results.
3. please insert linebreaks.

```suggestion
        if (!attribute.shareInfo) {
            const result = await client.attributes.deleteRepositoryAttribute(attribute.id);
            expect(result.isSuccess).toBe(true);
            continue;
        }

        if (attribute.content.owner === clientAddress) {
            const result = await client.attributes.deleteOwnSharedAttributeAndNotifyPeer(attribute.id);
            expect(result.isSuccess).toBe(true);
            continue;
        }

        if (attribute.content.owner !== clientAddress) {
            const result = await client.attributes.deletePeerSharedAttributeAndNotifyOwner(attribute.id);
            expect(result.isSuccess).toBe(true);
            continue;
        }

        if (attribute.shareInfo.thirdPartyAddress) {
            const result = await client.attributes.deleteThirdPartyRelationshipAttributeAndNotifyPeer(attribute.id);
            expect(result.isSuccess).toBe(true);
            continue;
        }

        throw new Error("No delete method called");
```